src/plugin.js

```diff
@@ -10,7 +10,7 @@ exports.register = function register(plugin, options, next) {
 
   _.forEach(options.white_list, value => {
     _.pullAll(value.cultures, [value.default]); // mutate array, removing default
-    value.cultures.unshift(value.default); // add default to front of array
+    value.cultures = _.concat([value.default], value.cultures); // add default to front of array
```

Comment:
I'm confused as to why this works. It should behave exactly the same, with the exception that the lodash approach you're switching to no longer mutates `value.default`.

```
> var arr = [1,2,3];
undefined
> arr.unshift(5);
4
> arr
[ 5, 1, 2, 3 ]
> arr.shift()
5
> arr
[ 1, 2, 3 ]
> _.concat([5], arr);
[ 5, 1, 2, 3 ]
```